### PR TITLE
LPS-84354 ensure that projectTemplateTokens are always processed

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
cc @lawrence-lee

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

File: modules/build.gradle [hunk 1]
```diff
@@ -114,6 +114,10 @@ gradle.beforeProject {
 				}
 
 				rename "gitignore", ".gitignore"
+
+				outputs.upToDateWhen {
+					false
+				}
```
Comment (petershin) [inline modules/build.gradle]: If order doesn't matter, please move `outputs. ..` before `rename` so its sorted alphabetically.

Comment (gamerson) [reply modules/build.gradle]: Willdo

Comment (gamerson) [reply modules/build.gradle]: pushed.

**Comment by petershin** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/62071

**Pull request closed without merging**